src/main/windows/main.ts

```diff
@@ -37,7 +37,7 @@ export function init(options: { hidden: boolean }) {
       x: initialBounds.x,
       y: initialBounds.y,
       webPreferences: {
-        nodeIntegration: false,
```

Comment:
If we bump electron then not with using this, rather doing the work that is required than weakening security to make it work